tests/integ/scala/test_async_job_suite.py

```diff
@@ -110,15 +110,14 @@ def test_async_to_pandas_common(session):
 @pytest.mark.skipif(IS_IN_STORED_PROC_LOCALFS, reason="Requires large result")
 @pytest.mark.skipif(not is_pandas_available, reason="Pandas is not available")
 def test_async_to_pandas_batches(session):
-    df = session.range(100000).cache_result()
+    df = session.range(100000).sort("id").cache_result()
```

Comment:
This test broke with the change I made to `src/snowflake/snowpark/_internal/server_connection.py`, because previously the exact same set of pandas batches are returned (from result reuse since) but now it is returning a different set. E.g. If there are 10 rows in total, `to_pandas_batches` could return a iteration of 3 rows, 4 rows, 3 rows OR an iteration of 5 rows, 5 rows. This doesn't break any user promise and I just rewrote the test to reflect this change.